skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -165,27 +163,92 @@ Beginning Microsoft Teams PowerShell version 2.6.2 onwards, the below updates ar
 - Phone
 - WindowsEmailAddress*
 
+*Supported filters* - Filtering has been limited to the below attributes
+
+- accountEnabled
+- ownerUrn
+- displayName
+- givenName
+- lineUri
```

Comment:
If I try to filter LineUri like this

` get-csonlineuser -Filter "lineUri -ne $null"`

I get this

>Microsoft.Teams.ConfigAPI.Cmdlets.internal\Search-CsUser : Filter couldn't be parsed

How to correctly filter by $null ?

Reply:
I am sorry, I actually used 
`get-csonlineuser -Filter "lineUri ne $null"`